### PR TITLE
Spack package for Ember

### PR DESCRIPTION
**Pull request by rvinaybharadwaj**

### CODE REVIEW

**Comment by sknigh** [comment]
Please spellcheck your doc-string.

**Comment by bhatele** [comment]
cc: @junghans

File: var/spack/repos/builtin/packages/ember/package.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    git      = "https://github.com/sstsimulator/ember.git"
+    url      = "https://github.com/sstsimulator/ember/archive/v1.0.0.tar.gz"
+
+    version('1.0.0', sha256='5b2a6b8055b46ab3ea2c7baabaf4d280d837bb7c21eba0c9f59e092c6fc1c4a6')
+
+    variant('halo3d', default=True, description='Halo3d motif')
+    variant('halo3d-26', default=False, description='Halo3d-26 motif')
+    variant('incast', default=False, description='Incast motif')
+    variant('pingpong', default=False, description='Pingpong motif')
+    variant('sweep3d', default=False, description='Sweep3d motif')
```
[the remark this answers is not in the document]
Comment (junghans) [reply var/spack/repos/builtin/packages/ember/package.py]: Agreed!

File: var/spack/repos/builtin/packages/ember/package.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    which are able to represent target, size, and data type of messages in
+    larger applications, libraries, and mini-apps.
+    """
+
+    homepage = "http://sst-simulator.org/SSTPages/SSTElementEmber/"
+    git      = "https://github.com/sstsimulator/ember.git"
+    url      = "https://github.com/sstsimulator/ember/archive/v1.0.0.tar.gz"
+
+    version('1.0.0', sha256='5b2a6b8055b46ab3ea2c7baabaf4d280d837bb7c21eba0c9f59e092c6fc1c4a6')
+
```
Comment (junghans) [inline var/spack/repos/builtin/packages/ember/package.py]: Add `tags = ['proxy-app', 'ecp-proxy-app']`.

**Pull request merged**